config/locales/en.yml

```diff
@@ -1665,6 +1665,7 @@ en:
       multiple_errors: "Multiple fields violated their constraints."
       parse_error: "The request body was neither empty, nor did it contain a single JSON object."
       writing_read_only_attributes: "You must not write a read-only attribute."
+      invalid_format: "Invalid format for property '%{property}': Expected format like '%{expected_format}', but got '%{actual}'."
```

Comment:
I was a bit startled when I accidentally inserted "2014-02-30" to test this because the format looks valid. I could also have sent "9999-99-99". Do you think it makes sense to also state what "Type" is required here? Then a user could look up the valid values in the API documentation as soon as the types are described there.